api/config/config_prod.json

```diff
@@ -137,5 +137,18 @@
   "offlineBatch": {
     "usersPerAuditTask": 20,
     "usersPerSynchronizeAccessTask": 50
+  },
+  "egressAlertRemediationPolicy": {
+    "enableJiraTicketing": true,
+    "notifyFromEmail": "egress@pmi-ops.org",
```

Comment:
Would `@pmi-ops.org` be confusing to users? Can we create one using` @researchallofus.org`?

Reply:
Good catch.. I agree.. Checking with the security team (they created the group). Probably I won't block on them to merge this PR since it may take more than a week.

Reply:
Will update the config once they've created the group. They agree something like `security-alerts@researchallofus.org` would be clearer.